content/docs/0.11.x/continuous_delivery/deployment_helm/index.md

```diff
@@ -25,17 +25,17 @@ $ kubectl get deployments -n sockshop-dev carts -owide
 
 ```
 NAME    READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts   1/1     1            1           56m   carts        docker.io/keptnexamples/carts:0.11.1   app=carts
```

Comment:
Please revert the change. The carts service is used in version `0.11.1` (see here: https://github.com/keptn-sockshop/carts/tree/release-0.11.0)